libmultilabel/nn/model_attentionxml.py

```diff
@@ -0,0 +1,73 @@
+import torch
+from torch import Tensor
+
+from .model import Model
+
+
+class PLTModel(Model):
+    def __init__(
+        self,
+        classes,
+        word_dict,
+        embed_vecs,
+        network,
+        loss_function="binary_cross_entropy_with_logits",
+        log_path=None,
+        **kwargs,
+    ):
+        super().__init__(
+            classes=classes,
+            word_dict=word_dict,
+            embed_vecs=embed_vecs,
+            network=network,
+            loss_function=loss_function,
+            log_path=log_path,
+            **kwargs,
+        )
+
+    def scatter_logits(
+        self,
+        logits: Tensor,
+        labels_selected: Tensor,
+        label_scores: Tensor,
+    ) -> Tensor:
+        """For each instance, we only have predictions on selected labels. This subroutine maps these predictions to
+        the whole label space. The scores of unsampled labels are set to 0."""
+        src = torch.sigmoid(logits.detach()) * label_scores
+        preds = torch.zeros(
+            labels_selected.size(0), len(self.classes) + 1, device=labels_selected.device, dtype=src.dtype
```

Comment:
explain why we need + 1.  I think you told me the reason but now I don't see why